FWCore/Concurrency/interface/WaitingTaskWithArenaSpawnHolder.h

```diff
@@ -0,0 +1,82 @@
+#ifndef FWCore_Concurrency_WaitingTaskWithArenaSpawnHolder_h
```

Comment:
Ultimately this class was not used (and given how TBB works lead to a deadlock) so can be removed.

Reply:
Me too